fitbenchmarking/fitbenchmarking
# Setting a minimum pandas version

#### Description of Work

Fixes #517 


#### Testing Instructions

1. Install using `pip install -e .`
2. check that output is created successfully

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
